src/Module/module.css

```diff
@@ -91,88 +32,35 @@ $moduleSpacing:  3rem;
         padding-bottom:     3rem;
         margin-bottom:      0;
 
-        &::before
-        {
-            transition: height 200ms ease-in-out, opacity 200ms ease-in-out;
-            opacity:    0;
-            height:     100%;
-        }
+        background: var(  --GS-11  );
+        transition: height 200ms ease-in-out, background-color 200ms ease-in-out;
+        height:     100%;
     }
 
-    &:hover > .header::before
+    &:hover > .header
     {
-        opacity:    1;
-        transition: height 200ms ease-in-out, opacity 200ms ease-in-out;
+        background: transparent;
+        transition: height 200ms ease-in-out, background-color 200ms ease-in-out;
     }
     .content
     {
         display:    none;
     }
 }
 
-.error
-{
-    & > .header::before
-    {
-        @extend %module-highlight;
-        opacity:            1;
-        background-color:   var( --AC-2 );
-    }
-}
-
 .moduleError
 {
-    &::before
+    & > .header
     {
-        content:            '';
-        position:           absolute;
-
-        left:               0;
-        top:                0;
-
-        height:             calc( 100% );
-        width:              0.6rem;
-
         background-color:   var( --AC-2 );
-        transition:         opacity 0.1s ease-in-out, opacity 200ms ease-in-out;
-
-    }
-}
-
-.subtleHighlight
-{
-    .header::before
-    {
-        display:            none;
-
-    }
-
-    &::before
-    {
-        @extend %module-highlight;
-        height:             100%;
-        background:         var( --PC-GREY );
-        opacity:            1;
-        left:               -3rem;
-
-    }
-
-    &.collapsed::before
-    {
-
-        opacity:            0;
-    }
-
-    &.collapsed:hover::before
-    {
-        transition:         height 200ms ease-in-out, opacity 200ms ease-in-out;
-        opacity:            1;
-
     }
 }
 
 .header
 {
+    background:         var(  --PC-WHITE  );
```

Comment:
Redundant to set background to white here (you’re overriding it to be `transparent` above anyways).